crates/mem/src/alloc.rs

```diff
@@ -70,7 +70,8 @@ impl<T: Pod> RawMem for Alloc<T> {
       let old_layout =
         Layout::array::<T>(old_cap).map_err(|_| Error::CapacityOverflow)?;
 
-      // SAFETY: reallocating with matching old_layout and growing to larger size
+      // SAFETY: reallocating with matching old_layout and growing to
+      // larger size
```

Comment:
avoid this long words, prefer to use realloc, grow/shrink
SAFETY comments must me oneline if it is short